javascript assignment 1/Strings.js

```diff
@@ -0,0 +1,6 @@
+const me = "Yadel"
+console.log(me.length);
+console.log(`Hello, my name is ${me}`)
+
+const myName = me.toUpperCase()
+console.log(`When my friends see me, they shout ${myName}!`)
```

Comment:
const me = "Yadel";
console.log(me.length); // Output: 5
console.log(`Hello, my name is ${me}`); // Output: Hello, my name is Yadel

const myName = me.toUpperCase();
console.log(`When my friends see me, they shout ${myName}!`); // Output: When my friends see me, they shout YADEL!